config/runtime.cfg

```diff
@@ -217,6 +217,42 @@
 #          Example:  COB_CURRENT_DATE "2026/03/16 16:40:52"
 #                    current_date YYYYMMDDHHMMSS+01:00
 
+# Environment name:  COB_PROF_FILE
+#   Parameter name:  prof_file
+#          Purpose:  to define where COBOL profiling output should go
+#             Type:  string where $$ is replaced by process id, $b executable
+#                    basename, $f executable filename, $d date yyyymmdd,
+#                    $t time hhmmss
+#          Default:  cob-prof-$b-$$-$d-$t.csv
+#          Example:  PROF_FILE  ${HOME}/$$-prof.csv
+
+# Environment name:  COB_PROF_ENABLE
+#   Parameter name:  prof_enable
+#          Purpose:  to enable profiling for modules compiled with profiling
+#             Type:  boolean
+#          Default:  false
```

Comment:
please add the note here about disabled physical cancel for each module that this applies to, too